client/src/js/analyses/components/NuVs/__tests__/BLASTInProgress.test.js

```diff
@@ -0,0 +1,13 @@
+import { RIDTiming } from "../BLASTInProgress";
+
+describe("<RIDTiming>", () => {
+    let props;
+    props = {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['props' is never reassigned. Use 'const' instead.](https://app.codacy.com/manual/virtool/virtool/pullRequest?prid=4813739)